From eecbd826b7cd61ac4bd6363f160cb66a5cde07a0 Mon Sep 17 00:00:00 2001
From: Lei Qian <lei.qian@amlogic.com>
Date: Wed, 19 Dec 2018 21:02:29 +0800
Subject: [PATCH] imageserver:remove imageserver to aosp only [2/2]

PD#SWPL-2263

Problem:
imageserver put into gtvs which will cause cts fail

Solution:
remove imageserver to aosp only

Verify:
Verify on Ampere

Change-Id: I071eeaca158f752c213f3e2cc13669f82e9f6a2a
Signed-off-by: Lei Qian <lei.qian@amlogic.com>
---
 ampere/manifest.xml          | 9 +++++++++
 braun/manifest.xml           | 9 +++++++++
 curie/manifest.xml           | 9 +++++++++
 darwin/manifest.xml          | 9 +++++++++
 einstein/manifest.xml        | 9 +++++++++
 franklin/manifest.xml        | 9 +++++++++
 galilei/manifest.xml         | 9 +++++++++
 hertz/manifest.xml           | 9 +++++++++
 t962_p321/manifest.xml       | 9 +++++++++
 t962_p321_1080p/manifest.xml | 9 +++++++++
 t962x2_skt/manifest.xml      | 9 +++++++++
 t962x2_x301/manifest.xml     | 9 +++++++++
 t962x_r314/manifest.xml      | 9 +++++++++
 u200/manifest.xml            | 9 +++++++++
 u221/manifest.xml            | 9 +++++++++
 15 files changed, 135 insertions(+)

diff --git a/ampere/manifest.xml b/ampere/manifest.xml
index c7a17a7..8e5f13f 100644
--- a/ampere/manifest.xml
+++ b/ampere/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/braun/manifest.xml b/braun/manifest.xml
index c1e61f5..0628660 100644
--- a/braun/manifest.xml
+++ b/braun/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/curie/manifest.xml b/curie/manifest.xml
index c7a17a7..8e5f13f 100644
--- a/curie/manifest.xml
+++ b/curie/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/darwin/manifest.xml b/darwin/manifest.xml
index 9c55127..9fa0e82 100644
--- a/darwin/manifest.xml
+++ b/darwin/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/einstein/manifest.xml b/einstein/manifest.xml
index b95ab30..71f219d 100644
--- a/einstein/manifest.xml
+++ b/einstein/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/franklin/manifest.xml b/franklin/manifest.xml
index f60bce7..8b5ba59 100644
--- a/franklin/manifest.xml
+++ b/franklin/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/galilei/manifest.xml b/galilei/manifest.xml
index 5f0d685..ec50c39 100644
--- a/galilei/manifest.xml
+++ b/galilei/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/hertz/manifest.xml b/hertz/manifest.xml
index f77fac1..f982d11 100644
--- a/hertz/manifest.xml
+++ b/hertz/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="2">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.0</version>
diff --git a/t962_p321/manifest.xml b/t962_p321/manifest.xml
index b95ab30..71f219d 100644
--- a/t962_p321/manifest.xml
+++ b/t962_p321/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/t962_p321_1080p/manifest.xml b/t962_p321_1080p/manifest.xml
index b95ab30..71f219d 100644
--- a/t962_p321_1080p/manifest.xml
+++ b/t962_p321_1080p/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/t962x2_skt/manifest.xml b/t962x2_skt/manifest.xml
index 9c55127..9fa0e82 100644
--- a/t962x2_skt/manifest.xml
+++ b/t962x2_skt/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/t962x2_x301/manifest.xml b/t962x2_x301/manifest.xml
index 9c55127..9fa0e82 100644
--- a/t962x2_x301/manifest.xml
+++ b/t962x2_x301/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/t962x_r314/manifest.xml b/t962x_r314/manifest.xml
index b95ab30..71f219d 100644
--- a/t962x_r314/manifest.xml
+++ b/t962x_r314/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/u200/manifest.xml b/u200/manifest.xml
index 3286dad..04bf3e8 100644
--- a/u200/manifest.xml
+++ b/u200/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
diff --git a/u221/manifest.xml b/u221/manifest.xml
index 32b5ae0..ad93bae 100644
--- a/u221/manifest.xml
+++ b/u221/manifest.xml
@@ -1,5 +1,14 @@
 <manifest version="1.0" type="device" target-level="3">
     <hal format="hidl">
+        <name>vendor.amlogic.hardware.imageserver</name>
+        <transport>hwbinder</transport>
+        <version>1.0</version>
+        <interface>
+            <name>IImageService</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <hal format="hidl">
         <name>android.hardware.configstore</name>
         <transport>hwbinder</transport>
         <version>1.1</version>
-- 
1.9.1

